server/src/metadata/migration-runner/migration-runner.module.ts

```diff
@@ -3,11 +3,11 @@ import { Module } from '@nestjs/common';
 import { DataSourceModule } from 'src/metadata/data-source/data-source.module';
 import { TenantMigrationModule } from 'src/metadata/tenant-migration/tenant-migration.module';
 
-import { MigrationGeneratorService } from './migration-generator.service';
+import { MigrationRunnerService } from './migration-runner.service';
```

Comment:
I think MigrationRunner is a better name since it was actually running the migration and not generating anything